public/static/docs/api-reference/get_url.md

```diff
@@ -0,0 +1,111 @@
+### `dvc.api.get_url(path, repo=None, rev=None, remote=None)`
+
+Returns the URL to the storage location of a data file or directory tracked by
+DVC.
+
+```py
+import dvc.api
+
+resource_url = dvc.api.get_url(
+    'get-started/data.xml',
+    repo='https://github.com/iterative/dataset-registry')
+
+# resource_url =
+# https://remote.dvc.org/dataset-registry/a3/04af...
+```
+
+All **parameter** types are
+[string](https://docs.python.org/3/library/stdtypes.html#text-sequence-type-str).
+
+## Description
+
+Returns the URL string of the physical location (in a
```

Comment:
do we need to mention `physical`?

Reply:
No, forgot to update this one before. Should be "storage"